### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by ptbrowne**
Various updates

- SVGr icons
- Update cozy-client and cozy-ui
- Update yarn scripts

And more... commits are self explanatory.

This is in preparation of future improvements for OIDC cozies.

### CODE REVIEW

File: src/components/CloudIcon/index.jsx [hunk 1]
```diff
@@ -2,16 +2,18 @@ import React from 'react'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 import './styles.css'
 
-const CloudIcon = props => {
+import CloudIcon from "cozy-ui/transpiled/react/Icons/Cloud";
```
Comment (Crash--) [inline src/components/CloudIcon/index.jsx]: weird to have " ";

File: src/components/CloudIcon/index.jsx [hunk 2]
```diff
@@ -2,16 +2,18 @@ import React from 'react'
...
+
+const PasswordsCloudIcon = props => {
   return (
     <Icon
-      icon="cloud"
+      icon={CloudIcon}
       size={16}
       color="var(--charcoalGrey)"
```
Comment (Crash--) [inline src/components/CloudIcon/index.jsx:12]: Should we use semantic var? ;)

**Review by Crash-- — APPROVED** [review]

File: src/components/CloudIcon/index.jsx (hunk 2 above)
Comment (Crash--) [reply src/components/CloudIcon/index.jsx:12]: Do we have a codemod to check the use of this kind of variable and replace it with primary/secondaryColor ?

File: src/components/CloudIcon/index.jsx (hunk 1 above)
Comment (ptbrowne) [reply src/components/CloudIcon/index.jsx]: I squashed the next lint commit into this one, this way we do not have the ";" (caused by the codemod).

File: src/components/CloudIcon/index.jsx (hunk 2 above)
Comment (ptbrowne) [reply src/components/CloudIcon/index.jsx:12]: We do not have a semantic variable for charcoalGrey I am afraid :/ I think here our best bet would be to use a condition on the theme mode 

```
const theme = useTheme()
const iconColor = theme.mode === 'light' ? theme.palette.grey[400] : 'white'
```

Comment (ptbrowne) [reply src/components/CloudIcon/index.jsx:12]: I will not do that in this PR ;)

**Pull request merged**